Vagrantfile

```diff
@@ -8,12 +8,14 @@ Vagrant.configure(2) do |config|
   config.vm.provision "bootstrap",
     type: "shell",
     inline: <<-SHELL
-      sudo apt-add-repository ppa:brightbox/ruby-ng
+      # add nodejs v12 repository
+      curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+
       sudo apt-get update
-      sudo apt-get install -yq ruby2.4 ruby2.4-dev
+      sudo apt-get install -yq ruby ruby-dev
```

Comment:
I think we're fine with this, but this specifically installs ruby 3, right?

Reply:
Whatever `ruby` and `ruby-dev` point to in Ubuntu's repositories. For `bionic` that's `2.5.1`, but newer releases use `3`.